tests/upgrade_path/test_upgrade_path.py

```diff
@@ -80,25 +64,52 @@ def test_upgrade_path(localhost, duthosts, rand_one_dut_hostname, ptfhost, upgra
 
             # Install target image
             logger.info("Upgrading to {}".format(to_image))
-            target_version = install_sonic(duthost, to_image, tbinfo)
-            test_params = ptf_params
-            test_params['target_version'] = target_version
-            test_params['reboot_type'] = get_reboot_command(duthost, upgrade_type)
-            prepare_testbed_ssh_keys(duthost, ptfhost, test_params['dut_username'])
-            log_file = "/tmp/advanced-reboot.ReloadTest.{}.log".format(datetime.now().strftime('%Y-%m-%d-%H:%M:%S'))
-            if test_params['reboot_type'] == reboot_ctrl_dict.get(REBOOT_TYPE_COLD).get("command"):
+            install_sonic(duthost, to_image, tbinfo)
+            if upgrade_type == REBOOT_TYPE_COLD: #reboot_ctrl_dict.get(REBOOT_TYPE_COLD).get("command"):
```

Comment:
What is the purpose of leaving the comment here?

Reply:
This was unintentional. Removed the comment now.